hack/tools.mk

```diff
@@ -30,6 +30,7 @@ HELM                       := $(TOOLS_BIN_DIR)/helm
 MOCKGEN                    := $(TOOLS_BIN_DIR)/mockgen
 OPENAPI_GEN                := $(TOOLS_BIN_DIR)/openapi-gen
 PROMTOOL                   := $(TOOLS_BIN_DIR)/promtool
+DOCFORGE                   := $(TOOLS_BIN_DIR)/docforge
```

Comment:
Binary is not used anywhere? Why do we need to install it?
Did you mean to download the `check-manifest` script here?